bundler/spec/commands/info_spec.rb

```diff
@@ -123,6 +123,22 @@
         expect(out).to_not include("Homepage:")
       end
     end
+
+    context "when gem has a reverse dependency" do
+      it "prints the details" do
+        bundle "info actionpack"
+
+        expect(out).to include("Reverse Dependencies: \n\t\trails (2.3.2) depends on actionpack (= 2.3.2)")
+      end
```

Comment:
Can we also test the "any version" case? I'm also suggesting it because although I'm not fully sure, I think requirements might never be empty but set to `[">= 0"]` instead.

Reply:
@deivid-rodriguez good call, you were right about that.  I've removed the "any version" line and added the test case that confirms `>= 0` 👍 

Back over to you.